pkg/plugins/network/modules.go

```diff
@@ -176,7 +176,7 @@ func deployCilium(d *DeployNetworkPluginModule) []task.Interface {
 		Hosts: d.Runtime.GetHostsByRole(common.Master),
 		Prepare: &prepare.PrepareCollection{
 			new(common.OnlyFirstMaster),
-			new(OldK8sVersion),
+			&OldK8sVersion{Not: true},
```

Comment:
I think `cilium` and `flannel` can remove this `prepare` named `OldK8sVersion`.

https://github.com/kubesphere/kubekey/blob/255c21afef3a25c132d54b91f9bebe271e0b80f5/pkg/plugins/network/modules.go#L142

Reply:
so do i need recommit a new PR with delete the `OldK8sVersion`  within `prepare` ?

Reply:
You can push a new commit to modify it and then rebase these commits.